runtime/lua/vim/treesitter/dev.lua

```diff
@@ -44,9 +44,25 @@ local TSTreeView = {}
 ---                  of injected language trees (see explanation above)
 ---@param tree TSP.Node[] Output table containing a list of tables each representing a node in the tree
 local function traverse(node, depth, lang, injections, tree)
+  if depth == 0 then
+    table.insert(tree, {
+      node = node,
+      depth = depth,
+      lang = lang,
+    })
```

Comment:
If we want to go a little bit further, though not mandatory: now that we want to include the root node, this implementation could be simplified by unifying all the three `table.insert(tree, { ... })` statements into just one call here. This traverse function should have additional argument `field` though.